docs/sources/reference/api/docker_remote_api.md

```diff
@@ -49,6 +49,12 @@ You can still call an old version of the API using
 `POST /containers/(id)/attach` and `POST /exec/(id)/start`
 
 **New!**
+Build now has support for `LABEL` command which can be used to add user data
```

Comment:
I think this block needs to move above `POST /containers/(id)/attach` and `POST /exec/(id)/start` and then the line `POST /build` should be added above it.  Looks like the format is API method/path followed by a paragraph describing the change to the API.